app-text/pdfgrep/pdfgrep-2.1.2.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+KEYWORDS="~amd64 ~x86"
```

Comment:
this line belongs to the block right above `IUSE`

Reply:
The order originates from the [live ebuild in my overlay](https://gitlab.com/Flow/flow-s-ebuilds/blob/master/app-text/pdfgrep/pdfgrep-9999.ebuild#L6). If not absolutely necessary, I'd like to keep it as is, because that makes it easier to distill the ebuild for the Gentoo repository out of the live ebuild.

Reply:
Please fix it - unless such a conditional block is actually part of the ebuild, order of https://github.com/gentoo/gentoo/blob/master/skel.ebuild shall be followed.